tools/common/schema/handler.go

```diff
@@ -108,3 +140,17 @@ func validateUpdateConfig(config *UpdateConfig) error {
 func flag(opt string) string {
 	return "(-" + opt + ")"
 }
+
+func schemaFileEnding(schemaName string) string {
+	if strings.Contains(schemaName, "cassandra") {
+		return ".cql"
+	}
+	return ".sql"
+}
+
+func validSchemaNames(dbName string) []string {
+	if dbName == "sql" {
+		return append(dbschemas.Paths("mysql"), dbschemas.Paths("postgresql")...)
+	}
+	return dbschemas.Paths(dbName)
+}
```

Comment:
Nit: this feels like something only the db implementation should care about

Reply:
to me it feels like the concept of a "schema name" is specific to the command line tool, since the concept of a "valid schema name" only exists because it's an option of the command line tool. Like, if the DB implementations are used elsewhere in our codebase, the function "db.ValidSchemaNames" would be irrelevant i think? and under that line of thinking, this function is something that the cli db tool is primarily concerned with, and putting it in the db implementation would pollute it with something that is only relevant because of the format of a specific command line option

Reply:
I'm fine with your approach, I was just actually curious if they were used elsewhere